### PR TITLE
[14.0][IMP] sale_commission: print partial and document total in report

### PR DESCRIPTION
**Pull request by HekkiMelody**

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@pedrobaeza anything against this? It's a useful change when printing settlement reports for many different settlement lines. WDYS?

**Comment by pedrobaeza** [comment]
Not sure about understanding the changes. Some screenshots of the before and after?

**Review by francesco-ooops — APPROVED** [review]
Sure @pedrobaeza, on the left current version, on the right this PR

image

In red: "total" of a single settlement was printed on each page related to that settlement, making it confusing when printing multiple settlements at once.

Now "total" is printed only at the end of a single settlement.

In blue: The total of all settlements printed in the report is explicit.

File: sale_commission/views/report_settlement_templates.xml [hunk 1]
```diff
@@ -2,9 +2,22 @@
...
+            <t
+                t-set="show_doc_total"
+                t-value="len(docs) > 1 and len(docs.mapped('currency_id')) == 1"
+            />
             <t t-foreach="docs" t-as="o">
                 <t t-call="web.internal_layout">
                     <div class="page">
+                        <t t-if="show_doc_total">
+                            <div class="text-right">
+                                <strong>Document total:</strong>
```
Comment (pedrobaeza) [inline sale_commission/views/report_settlement_templates.xml]: `Document total` wording seems a bit confusing. Better to put it `Global amount` or `Global commission amount`.

Comment (HekkiMelody) [reply sale_commission/views/report_settlement_templates.xml]: Thank you for the review. Changed the wording as you suggested

**Review by francesco-ooops — APPROVED** [review]

**Comment by francesco-ooops** [comment]
Hi @pedrobaeza can this be merged? It's quite a minor change.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

As always, please fw-port it

**Pull request merged**